pages/mesosphere/dcos/2.2/release-notes/2.2/index.md

```diff
@@ -27,20 +27,59 @@ This release provides new features and enhancements to improve the user experien
 DC/OS's Universal Container Runtime (UCR) now supports external volumes provided via the Container Storage Interface (CSI). Storage providers which integrate with the CSI specification provide plugins which DC/OS users may install into their cluster. These plugins allow volumes backed by that provider to be attached to task containers. This initial release of CSI support in DC/OS 2.2 has some caveats. For more information, see the [CSI](/mesosphere/dcos/2.2/storage/external-storage/csi/).
 
 # Jobs with dependencies
-Metronome based jobs can have one more dependencies specified; a job will only be run when all of its dependencies have successfully run. This capability allows users to natively setup DAG based workflows in DC/OS. For more information, see [dependencies](mesosphere/dcos/2.2/deploying-jobs/quickstart#dependencies).
+Metronome based jobs can have one more dependencies specified; a job will only be run when all of its dependencies have successfully run. This capability allows users to natively setup DAG-based workflows in DC/OS. For more information, see [dependencies](mesosphere/dcos/2.2/deploying-jobs/quickstart#dependencies).
 
 # Custom CA certificate rotation
-DC/OS now allows an operator to rotate the custom CA certificates by simply updating configuration settings during an upgrade. This feature ensures all the services that are using the custom CA based certificates are automatically updated after an upgrade.
+DC/OS now allows an operator to rotate the custom CA certificates by simply updating configuration settings during an upgrade. This feature ensures all the services that are using the custom CA based certificates are automatically updated after an upgrade. 
 
 # Breaking changes
 
 # Component Versions
 DC/OS 2.2.0 includes the following component version updates:
 
+- Marathon 1.11.16
+
```

Comment:
Let's update this:

-- Apache® Mesos® 1.11.0-dev
-- Marathon 1.11.23
-- Metronome 0.6.68
-- DC/OS UI  6.1.16
-- CockroachDB 19.1.11
-- Etcd 3.4.9

Reply:
done